lib/habits/achievements/achievements.ex

```diff
@@ -3,7 +3,7 @@ defmodule Habits.Achievements do
   The Achievements context.
   """
 
-  alias HabitsWeb.{Habit}
+  alias Habits.Habits.Habit
```

Comment:
I debated calling this context `Habits.Core`. I might still, since `Habits.Habits.Habit` is a little silly. Though to solve that I could also rename the repo.